<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="rootDirectoyPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Directory</string>
    </property>
   </widget>
   <widget class="QListWidget" name="folderListView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>591</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="fileListView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>591</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getTreeSizePushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Tree Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="foldersLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Folders:</string>
    </property>
   </widget>
   <widget class="QLabel" name="filesLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Files:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backPushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="treeSizeLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
